doc/server_API.md

```diff
@@ -110,6 +97,7 @@ On success:
 | 200(OK) | Success |
 | 500 (Internal Server Error) | Server error|
 | 201 (Created) | Created whitelist |
+| 303 (See Other) | Whitelist with such name already exists |
```

Comment:
Codes 3XX are for redirects, use code that starts with 4 (f.e. 454).

See codes here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status